ansible/eda-server
# [AAP-12134] Add a new RQ to manage non-rulebook tasks

## Problem Definition

Currently we have a fixed number of workers defined at installation time based on the hardware resources. With the current implementation each worker takes care of one activation. If the user creates enough activations, all the workers can be busy with activations preventing other async tasks like import projects.

## Solution

Implement a different queue for activations and ensure not all the workers are subscribed to that queue to ensure that there are always available workers for the rest of async tasks, for example project import.


## Testing 

1. Create multiple activations (preferably more than the number of workers available)
2. Start all activations and import a new project at the same time
3. All tasks should go through in the order they come in without any issues (e.g. activations getting stuck on starting, etc.)

JIRA ticket: [AAP-12134](https://issues.redhat.com/browse/AAP-12134)

NOTE: This PR should only be merged in sync with productization and after this PR https://github.com/ansible/eda-server/pull/320 gets merged.
